application/templates/pages/accessibility-statement.md

```
@@ -1,101 +1,149 @@
 
-#### 3.2.2: On Input
+### Non-accessible content
 
-Search functionality on the [dataset index](/dataset/) and [organisation index](/organisation/) performs an update to page content by filtering a list of links. It is a change of content that could be a 'change of context' because the viewport changes height and potentially scroll position as the number list items increases or decreases.
+The content listed below is non-accessible for the following reasons.
 
-#### 3.3.1: Error Identification
+## Non-compliance with the accessibility regulations 
 
-In the 'search' page. There is functionality to search for a dataset to facet search results by. However there is no visual feedback if the user types a search with no results, the content is visually hidden.
+#### Information, structure and relationships (WCAG 1.3.1) 
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)